.github/workflows/scheduled-gcp-experiment-pipeline.yml

```diff
@@ -0,0 +1,183 @@
+          VM_INSTANCE_NAMES: "${{ secrets.INSTANCE_ONE }},${{ secrets.INSTANCE_TWO }}"
+          INSTANCE_ZONES: "${{ secrets.ZONE }},${{ secrets.ZONE }}"
+          GO_EXPERIMENT_IMAGE: "${{ github.event.inputs.goExperimentImage }}"
+          EXPERIMENT_IMAGE_PULL_POLICY: "${{ github.event.inputs.experimentImagePullPolicy }}"
+          CHAOS_NAMESPACE: "${{ github.event.inputs.chaosNamespace }}"
+        run: make gcp-vm-instance-stop
+
+      - name: Delete target GCP VM Instances
+        if: always()
+        run: |
+          gcloud compute instances delete ${{ secrets.INSTANCE_ONE }} ${{ secrets.INSTANCE_TWO }} \
+          --zone=${{ secrets.ZONE }} \
+          --quiet
+
+  GCP_VM_Disk_Loss:
+    runs-on: ubuntu-latest
+    needs: GCP_VM_Instance_Stop
+    steps:
+      
+      #Install and configure a k3s cluster
+      - name: Installing Prerequisites (K3S Cluster)
+        env: 
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+        run: |
+          curl -sfL https://get.k3s.io | sh -s - --docker --write-kubeconfig-mode 664
+          kubectl wait node --all --for condition=ready --timeout=90s
+          mkdir -p $HOME/.kube && cat /etc/rancher/k3s/k3s.yaml > $HOME/.kube/config          
+          kubectl get nodes
+          
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.14'
+
+      - name: Create Kubernetes secret for gcp experiment
+        if: always()
+        env:
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+        run: |
+          cat <<EOF | kubectl apply -f -
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: cloud-secret
+          type: Opaque
+          stringData:
+            type: service_account
+            project_id: ${{ secrets.GCP_PROJECT_ID }}
+            private_key_id: ${{ secrets.PRIVATE_KEY_ID }}
+            private_key: ${{ secrets.PRIVATE_KEY }}
+            client_email: ${{ secrets.CLIENT_EMAIL }}
+            client_id: ${{ secrets.CLIENT_ID }}
+            auth_uri: ${{ secrets.AUTH_URI }}
+            token_uri: ${{ secrets.TOKEN_URI }}
+            auth_provider_x509_cert_url: ${{ secrets.AUTH_PROVIDER_CERT_URL }}
+            client_x509_cert_url: ${{ secrets.CLIENT_CERT_URL }}
+          EOF
+
+      - name: Set up Google Cloud SDK
+        if: always()
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - name: Create a GCP VM Instance with target Disk Volumes
+        if: always()
+        run: |
+          gcloud compute instances create ${{ secrets.INSTANCE_ONE }} \
+          --machine-type=${{ secrets.MACHINE_TYPE }} \
+          --zone=${{ secrets.ZONE }} \
+          --create-disk name=${{ secrets.DISK_ONE }},size=1GB,device-name=${{ secrets.DISK_ONE }} \
```

Comment:
I think the disk can also be created in the previous step. So we'll have two more steps.
1. create disk
2. create instance
3. attach the disk to the instance.